week1/homework/src/server.js

```diff
@@ -10,6 +10,33 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    const giveResult = (statusCode, object) => {
+      response.writeHead(statusCode, { 'Content-Type': 'application/json' });
+      response.write(JSON.stringify(object));
+    };
+
+    switch (request.url) {
+      default: giveResult(404, { error: `Not found` });
```

Comment:
for a best practice and more legibility I would go for keeping this `default` case at the end of my code.

Reply:
My code is a reflection of how I thought while I writing it. I started with the default situation, then started to add  the others possibility one buy one.  after finishing I searched again to know if it is normaal to keep the default on the top. And the answer was: yes it is possible,  so I did not change it. 
But on the other hand I agree with what you mentioned.